models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -40,7 +40,7 @@ class ReverseObjectRelation extends ManyToManyObjectRelation
      *
      * @var string
      */
-    public string $ownerClassName;
+    public ?string $ownerClassName;
```

Comment:
```suggestion
    public ?string $ownerClassName = null;
```

Reply:
No it isn't

Reply:
https://3v4l.org/fevpm

Reply:
Untyped properties are `null` if not initialized, whereas typed properties are "not initialized".

Reply:
Ahhh, thanks for explanation, will adjust the code.